plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -1226,7 +1227,7 @@ private boolean checkOvsNetwork(final String networkName) {
             return true;
         }
 
-        final Script command = new Script("/bin/sh", _timeout);
+        final Script command = new Script("/bin/sh", _timeout.getMillis());
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.